h/api/search/test/config_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import itertools
+import re
+import urllib
+
+from h.api.search.config import ANNOTATION_ANALYSIS
+
+
+def test_strip_scheme_char_filter():
```

Comment:
Your guess is as good as mine. This is just moved code, and `git log -S test_strip_scheme_char_filter` reveals the not-altogether-enlightening:

```
commit b0ed4a4
Author: Randall Leeds <tilgovi@hypothes.is>
Date:   2015-04-28 16:05:46 -0700

    Test elasticsearch filters and tokenizers
```